terrain/voxel_lod_terrain.cpp

```diff
@@ -1198,12 +1198,12 @@ void VoxelLodTerrain::_process() {
 			const VoxelMesher::Output mesh_data = ob.data.smooth_surfaces;
 
 			// TODO Allow multiple collision surfaces
-			Array collidable_surface;
+			Vector<Array*> collidable_surface;
```

Comment:
In fact there is probably no need to create an extra vector, you could use `mesh_data.surfaces` directly? Then no need for the last argument either